packages/block-editor/src/components/block-list/block-selection-button.js

```diff
@@ -63,6 +73,13 @@ function BlockSelectionButton( { clientId, rootClientId, ...props } ) {
 	// Focus the breadcrumb in navigation mode.
 	useEffect( () => {
 		ref.current.focus();
+
+		// NVDA on windows suffers from a bug where focus changes are not announced properly
+		// See WordPress/gutenberg#24121 and nvaccess/nvda#5825 for more details
+		// To solve it we announce the focus change manually.
+		if ( isWindows() ) {
+			speak( label );
```

Comment:
I also wondered whether I should announce something like `label + ', button'`

Reply:
I think label is good enough.